broker/src/main/java/io/zeebe/broker/clustering/topology/TopologyManagerImpl.java

```diff
@@ -189,9 +189,22 @@ private BrokerInfo readBrokerInfo(final Member eventSource) {
     final BrokerInfo brokerInfo = BrokerInfo.fromProperties(eventSource.properties());
     if (brokerInfo != null && !isStaticConfigValid(brokerInfo)) {
       LOG.error(
-          "Static configuration of node {} differs from local node {}",
+          "Static configuration of node {} differs from local node {}.",
           eventSource.id(),
           atomix.getMembershipService().getLocalMember().id());
+
+          LOG.error(
```

Comment:
Since this is not really a different error than the error log statement at line 191, I'd prefer to see these log statements combined into a single statement. Otherwise our error tooling for pick it up as 2 separate errors.